.github/workflows/CI.yml

```diff
@@ -27,7 +36,7 @@ jobs:
         python -m pip install --upgrade pip
         pip install -r requirements-dev.txt
     - name: Install dependencies for pypy
-      if: matrix.python-version == 'pypy-3.7'
+      if: matrix.python-version == 'pypy-3.9'
```

Comment:
Should something like the same comment as somewhere further on be added here? 
` # Recognize for the moment this will NOT run as we aren't using CPython 3.9`